Library/Formula/git-plus.rb

```diff
@@ -0,0 +1,25 @@
+class GitPlus < Formula
+  homepage "https://github.com/tkrajina/git-plus"
+  head "https://github.com/tkrajina/git-plus.git"
+  url "https://github.com/tkrajina/git-plus/archive/v0.1.0.tar.gz"
+  sha256 "d7076ed3fd8e6e4bc2b93b9880adf58bea59b0f6fe95e05053f00247ae8cb609"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  def install
+    site_packages = libexec/"lib/python2.7/site-packages"
+    libexec_bin = libexec/"bin"
+
+    ENV.prepend_create_path("PYTHONPATH", site_packages)
+
+    site_packages.install("gitutils.py")
+    libexec_bin.install(Dir["git-*"])
+
+    bin.install(Dir[libexec_bin/"*"])
+    bin.env_script_all_files(libexec_bin, :PYTHONPATH => ENV["PYTHONPATH"])
+  end
+
+  test do
+    shell_output("git multi")
```

Comment:
See [this comment](https://github.com/Homebrew/homebrew/pull/37985#issuecomment-84963044).

We can be thorough though if you're feeling adventurous; Something like this will work:

```ruby
  test do
    mkdir "testme" do
      system "git", "init"
      touch "README"
      system "git", "add", "README"
      system "git", "commit", "-m", "testing"
      rm "README"
    end

    assert_match /D README/, shell_output("#{bin}/git-multi")
  end
```